api/receiver.py

```diff
@@ -673,6 +675,27 @@ def start_receiving(self):
                 except Exception:
                     pass
 
+    def run_script(self, arguments):
+        module_name = arguments[0]
+        script_name = arguments[1]
+        body_input = json.loads(arguments[2])
+        try:
+            with open(self.temp_dir + "/run-script-api-stderr.txt", "w") as f:
```

Comment:
I don t think we need to open this file here - it was used only when we were running the sub-process which is not this case

Reply:
File opening removed